File: Beecrowd/3-Strings/1234-Sentença_Dançante-3/1234.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    char input[55];
    while (fgets(input, sizeof(input), stdin)) {
        int size = strlen(input);
        int axu = 0;

        for (int i = 0; i < size; i++) {
            if (input[i] == ' ') {
                printf(" ");
                continue;
            }

            axu++; 
            
            if (axu % 2 == 1) {
                if (isupper(input[i])) {
                    printf("%c", input[i]);  
                } else {
                    printf("%c", toupper(input[i])); 
                }
            } else {
                if (islower(input[i])) {
                    printf("%c", input[i]);  
                } else {
                    printf("%c", tolower(input[i]));
                }
            }
        }
    }

    return 0;
}

File: Beecrowd/3-Strings/1234-Sentença_Dançante-3/1234.cs
using System;

class Solution {
    static void Main() {
        string input;
        while ((input = Console.ReadLine()) != null) {
            int axu = 0;

            foreach (char c in input) {
                if (c == ' ') {
                    Console.Write(" ");
                    continue;
                }

                axu++;

                if (axu % 2 == 1) {
                    Console.Write(char.ToUpper(c));
                } else {
                    Console.Write(char.ToLower(c));
                }
            }
            Console.WriteLine();
        }
    }
}
